app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
According to the checklist, you should ensure that the `cp` input command is correct. However, your function does not handle incorrect command inputs. You might want to add error handling for cases when the command is not formatted correctly.